shuttheboxgame.py

```diff
@@ -46,21 +51,47 @@ def shutthebox_v1(tiles):
         old_tiles = tiles.copy()
         strat.__startlow__()
```

Comment:
This function returns a variable but nothing is being updated from the return so it is like nothing is happening here...

Reply:
I think I fixed this! Its return now updates some variables